packages/aws-rfdk/lib/deadline/lib/usage-based-licensing.ts

```diff
@@ -594,10 +593,12 @@ export class UsageBasedLicensing extends Construct implements IGrantable {
   public grantPortAccess(workerFleet: IWorkerFleet, licenses: UsageBasedLicense[]) {
     licenses.forEach(license => {
       license.ports.forEach(port => {
-        this.connections.allowFrom(workerFleet, port);
+        // this.connections.allowFrom(workerFleet, port);
```

Comment:
nit: we can remove this commented-out old/broken implementation. same goes for below.

Reply:
Yup, yup. That's why I marked the PR as a Draft. ;-) 
I also still have to validate the changes to the example apps.

Reply:
oops. missed the draft marker - sorry